packages/cdk8s-cli/lib/config.ts

```diff
@@ -4,11 +4,16 @@ import { Language } from './import/base';
 
 const CONFIG_FILE = 'cdk8s.yaml';
 
+export interface Cdk8sImport {
+  moduleName?: string;
```

Comment:
Rename to `name` and `source` (it’s not always a file... could be url or k8s...)